tests/foreman/ui/test_host.py

```diff
@@ -2108,6 +2110,39 @@ def test_all_hosts_bulk_build_management(target_sat, function_org, function_loca
         )
 
 
+@pytest.mark.tier2
+def test_bootc_booted_container_images(target_sat, bootc_host, function_ak_with_cv, function_org):
+    """Create a bootc host, and read it's information via the Booted Container Images UI
+
+    :id: c15f02a2-05e0-447a-bbcc-aace08d40d1a
+
+    :expectedresults: Booted Container Images contains the correct information for a given booted image
+
+    :CaseComponent:Hosts-Content
```

Comment:
Please confirm the component. I think this test should be part of Hosts component, as it just a registering host and validating its information and nothing related to content manipulation.
```suggestion
    :CaseComponent:Hosts
```